sources/advertisements/VRM New Core/Controllers/StartAdProcessingController.swift

```diff
@@ -5,79 +5,120 @@ import PlayerCore
 
 final class StartAdProcessingController {
```

Comment:
No sense in having this controller - just use preroll and midroll controller with separate logic.